Source/SmallBasic.Editor/Libraries/Graphics/TurtleGraphicsObject.cs

```diff
@@ -13,15 +14,24 @@ internal sealed class TurtleGraphicsObject : BaseGraphicsObject
         public TurtleGraphicsObject(GraphicsWindowStyles styles)
             : base(styles)
         {
+            this.Width = 48;
+            this.Height = 61;
         }
 
+        public decimal Width { get; set; }
```

Comment:
Can we use auto-getters here, instead of assigning in the constructor?

```suggestion
        public decimal Width => 48;
```

Reason is:
1) These properties are not writable. It shouldn't be possible to change these values at runtime.
2) Enables .net to inline these calls whenever possible, which leads to better performance.